tests/unit/customizations/s3/test_results.py

```diff
@@ -277,6 +291,54 @@ def test_multiple_progress_results(self):
             num_results * bytes_transferred
         )
 
+    def test_progress_result_with_no_known_transfer_size(self):
+        self.result_recorder(
+            QueuedResult(
+                transfer_type=self.transfer_type, src=self.src,
+                dest=self.dest, total_transfer_size=None
+            )
+        )
+
+        bytes_transferred = 1024 * 1024  # 1MB
```

Comment:
pep8: "Inline comments are unnecessary and in fact distracting if they state the obvious."